src/clj/chronograph/db/user.clj

```diff
@@ -35,3 +35,11 @@
    (first (sql/query tx
                      ["SELECT * FROM users WHERE id=?" user-id]
                      db/sql-opts))))
+
+(defn find-by-org-id
+  ([tx organization-id]
```

Comment:
This seems to have only one arity. Written as multi-arity fn.

Reply:
Done